dotcom-rendering/src/lib/edition.test.ts

```diff
@@ -0,0 +1,52 @@
+import {
+	editionalisedPages,
+	editionList,
+	isEditionalisedPage,
+	isNetworkFront,
+} from './edition';
+
+const everyNetworkFront = editionList.map((edition) => edition.pageId);
+
+const everyEditionalisedPage = editionList
+	.map((edition) =>
+		editionalisedPages.map((page) => `${edition.pageId}/${page}`),
+	)
+	.flat();
+
+describe('is network front', () => {
+	it('returns true if pageId is a network front', () => {
+		expect(everyNetworkFront.every((page) => isNetworkFront(page))).toBe(
+			true,
+		);
+	});
+	it('returns false if pageId is NOT a network front', () => {
+		expect(
+			everyEditionalisedPage.every((page) => isNetworkFront(page)),
```

Comment:
You might find the following more succint and explicit?

```suggestion
			everyEditionalisedPage.every(isNetworkFront),
```